xml/metering-metering_overview.xml

```diff
@@ -8,11 +8,21 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="ceilo_metering_overview">
  <title>Metering Service (Ceilometer) Overview</title>
  <para>
-  The &productname; Metering service is based on the OpenStack Ceilometer
-  API. The &productname; metering service collects and provides access to OpenStack
+  The &productname; metering service collects and provides access to OpenStack
   usage data that can be used for billing reporting such as showback, and
   chargeback. The metering service can also provide general usage reporting.
+  Ceilometer acts as the central collection and data access service to
+  the meters provided by all the OpenStack services. The data collected 
+  is available both through the Monasca API and the Ceilometer V2 API.
  </para>
+ <important>
+   <para>
+     Ceilometer V2 API has been deprecated in Pike release upstream. 
+     Although the Ceilometer V2 API is still available with &kw-hos;, to prepare 
+     for eventual removal of  Ceilometer V2 API in next release we recommend that
+     users should switch to the Monasca API to access data.
```

Comment:
You can drop the word 'should', it sounds a bit redundant with 'recommend'. :)

Reply:
Thx! done